libs/utils/utils.js

```diff
@@ -157,10 +157,9 @@ export const [setConfig, getConfig] = (() => {
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
-        let dir = getMetadata('content-direction') || config.locale.dir;
-        if (!dir && config.locale.ietf) {
-          dir = dir || new Intl.Locale(config.locale.ietf)?.textInfo?.direction;
-        }
+        const dir = getMetadata('content-direction')
+          || config.locale.dir
+          || (config.locale.ietf && (new Intl.Locale(config.locale.ietf)?.textInfo?.direction));
         document.documentElement.setAttribute('dir', dir || 'ltr');
```

Comment:
Since we already chain everything to `const dir`, I'd also add 'ltr' onto that instead of having a second conditional assignment within setAttribute.